mava/configs/network/ff_retention.yaml

```
@@ -0,0 +1,12 @@
+# ---  Retention for Non-memory Sable ---
+net_config:
+  n_block: 1 # Number of blocks
+  embed_dim: 64 # Embedding dimension
+  n_head: 1 # Number of heads
```

As in MAT I think just remove the `net_config` key

```suggestion
n_block: 1 # Number of blocks
embed_dim: 64 # Embedding dimension
n_head: 1 # Number of heads
```